decidim-participatory_processes/lib/decidim/participatory_processes/test/factories.rb

```diff
@@ -31,6 +31,7 @@
     private_space { false }
     start_date { Date.current }
     end_date { 2.months.from_now }
+    area { nil }
```

Comment:
I don't think this is needed

Reply:
Yes, `{ nil }` is needed because otherwise a bunch of tests using this factory fail.

Reply:
I mean I think you can remove the whole line 😄